bin/tf

```diff
@@ -87,6 +87,16 @@ def exec_tf_command(
     if command == "init":
         shutil.rmtree(os.path.join(path, ".terraform"), ignore_errors=True)
 
+    if command == "import":
```

Comment:
Do you have a check verifying that it is actually en existing state file, or will we get this message for every import command?

Reply:
For every import command. I'm not sure if it's worth the effort to make that kind of check. You'd have to check if it's merged, essentially.

Reply:
I believe if path exists in pipeline is a good indicator that state file is not new.

Reply:
But we're removing pipelines.

Reply:
There is a scenario of running some sort of script to import multiple resources in state file.

Reply:
You could use `yes`

Reply:
👍